config/runtimes/kserve-torchserveserver-gpu.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-torchserveserver-gpu
```

Comment:
I think the gpu runtime can be consolidated, it is the image tag which is different.

for cpu image it is `0.5.0` while gpu image tag is `0.5.0-gpu`